### DISCUSSION TITLE
Undefined symbols for architecture arm64

### DISCUSSION THREAD

**Discussion by sukantasamanta96** [community/community#65014] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

While i am trying to Archive a react native project, I am getting an error that is "Undefined symbols for architecture arm64".

Configuration is :- 
     React Native  :- "0.71.6"
     React :- "18.2.0"
     Xcode Version :- 14.2

The error is in the bellow :-

<img width="1398" alt="Screenshot 2023-08-26 at 1 47 52 PM" src="https://github.com/community/community/assets/68897331/a318b07e-90cf-4675-aa10-6a28639ab36d">

Please help me to solve this problem.

**Comment by arafajs** [comment]
1. Select your project name in the project navigator
2. Select your target that is having this issue
3. Find "Valid Architectures" under "Build Setting" > "Architectures"
4. Remove "arm64 & arm7" from "Valid Architectures" from the relevant scheme
5. Compile your project with release mode

**Reply by arafajs** [reply]
If not working, then Make sure JavaScriptCore.framework is added to Link Binary With Libraries.

**Reply by sukantasamanta96** [reply]
Thanks @arafajs for your reply,

I don'i find the "Valid Architectures" option in  "Build Setting" > "Architectures".

<img width="1391" alt="Screenshot 2023-08-26 at 3 12 37 PM" src="https://github.com/community/community/assets/68897331/4015012f-5f26-407d-a4e8-c3dc49477cfa">

**And also follow the 2nd step** 

<img width="1396" alt="Screenshot 2023-08-26 at 3 15 54 PM" src="https://github.com/community/community/assets/68897331/55ebbfdb-1d14-49b9-b996-182b2636deb3">


But I am getting same error during the Archive time.

**Reply by arafajs** [reply]
- In the search bar, type "Excluded Architectures".
- In the list of build settings, double-click on the Excluded Architectures build setting.
- In the Value field, enter the architectures that you want to exclude from the build.
- Click on the Apply button.

**Reply by sukantasamanta96** [reply]
Add "arm64" for only simulator and for Debug and Release added nothing.

<img width="1431" alt="Screenshot 2023-08-26 at 4 31 41 PM" src="https://github.com/community/community/assets/68897331/a9a53cb4-227b-4841-a706-422d1144cfd0">

Same issue is occurring.
